docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - docker-conf/postgres.env
       - docker-conf/redis.env
     ports:
-      - "443:443"
```

Comment:
I think that this should probably be this way, be because it will make it possible to go to `https://localhost` to reach the webui. Does that make sense?

Reply:
Fair enough, my complaint is more specific to Docker I guess.